### ISSUE TITLE
Add support for EXCLUDING attachment extensions

### ISSUE THREAD

**Issue by situfang** [husjon/obsidian-file-cleaner-redux#36]
Great plugin.
Being able to configure attachment extensions that need to be cleaned is a nice feature. From the perspective of my usage, I would like to clean up all useless extensions (configure ".*" directly), but exclude specified suffix names, such as .mdenc (which is used by another encryption plug-in and should not be cleaned up).
Thank you!

**Comment by husjon** [comment]
Hm, so if I understand you correctly, you'd like inverse matching.

F.ex if you have the following files:
```
file.md
file.jpg
file.png
file.pdf
file.mdenc
```

And you set your filter to be:
```
.md
.mdenc
```

You want it to **all** files except `.md`and `.mdenc` to removed?

So, the remaining files would be:
```
file.md
file.mdenc
```

**Comment by situfang** [comment]
Yes, that's basically what I expected. But it would be more reasonable not to clean up the .md file for the plugin.
I think .md does not need to be configured here, otherwise the user may delete them by mistake.

**Comment by husjon** [comment]
Thanks for the feedback.
The only reason why the markdown files would be cleaned up is if they are empty so they wouldn't be removed by mistake :)

I'll have a look at how to implement it and get back to you when it's ready.

**Comment by husjon** [comment]
Hey @situfang, just a heads up that I've implemented this and it will be part of v0.9.0 releasing soon.
This issue will be closed automatically when the PR has been merged.
